tests/unit/bases/test_almalinux.py

```diff
@@ -843,72 +829,60 @@ def test_pre_setup_snapd_failures(fake_process, fake_executor, fail_index):
 
 
 @pytest.mark.usefixtures("stub_verify_network")
-def test_setup_snapd_failures(fake_process, fake_executor):
+@pytest.mark.parametrize("fail_index", list(range(0, 7)))
+def test_setup_snapd_failures(fake_process, fake_executor, fail_index, mocker):
```

Comment:
Diff is pretty worthless here. The change here is that 6 snapd calls were moved from `post_setup_snapd` (called during `setup` and `warmup`) to `setup_snapd` (called during `setup`).

Reply:
The diff is now readable